tap_db2/__init__.py

```diff
@@ -56,83 +55,134 @@
 logger = logging.getLogger(__name__)
 logger.setLevel(logging.INFO)
 
+# Define data types
+
+# Full list
+#BIGINT - i
+#BLOB - ignore for now
+#CHARACTER - s
+#CLOB - ignore for now
+#DATE - d
+#DECIMAL - f
+#DOUBLE - f
+#INTEGER - i
+#SMALLINT - i
+#TIMESTAMP - d
+#VARCHAR - s
+#XML - s
+
 STRING_TYPES = set(
```

Comment:
From a completeness perspective, I wonder if we should add in support for other potential string types and clobs. There is support vargraphic as a datatype which I believe supports multi-characters which may be required for Macrons.

https://www.ibm.com/docs/en/db2-for-zos/11?topic=columns-string-data-types.

Perhaps there could be value by adding these additional datatypes?

For really large datatypes e.g. clob, you can convert them to a string to transfer some of the data. For example, tap-oracle has support here : https://github.com/s7clarke10/pipelinewise-tap-oracle/blob/master/tap_oracle/__init__.py#L122-L138 and here https://github.com/s7clarke10/pipelinewise-tap-oracle/blob/master/tap_oracle/sync_strategies/common.py#L40-L42 .

Reply:
Will move onto this in a new feature